### ISSUE TITLE
Nextcloud Notes for Android - Images are not displayed

### ISSUE THREAD

**Issue by stefan-franz** [nextcloud/notes-android#1877]
In Notes in Nextcloud (access with Firefox browser on Win) it's possible to add images in Notes and it works. But in the Notes Android App these images are not displayed.

**Comment by gautierlabarre** [comment]
Hello, I too need this feature :( any idea on when this might be implemented ? 🙏🏻

**Comment by hgdrn** [comment]
I'm here because I thought I'm doing anything wrong. The app says "use absolute paths or complete URLs". But: the app will only show images with a complete URL, Nextcloud Web shows images with an absolute path, too.

This works both in Android and Web:
![foobar.jpg](https://$DOMAIN/remote.php/dav/files/$USER/Notizen/Foo/foobar.jpg)

This works only in Web: but not in the Android App:
![foobar.jpg](/Notizen/Foo/foobar.jpg)

Relative paths neither work in the Android App.

If I edit a note in Nextcloud Web and paste an image from the clipboard the image data is saved as an attachement to a ".attachments.$ID" subfolder, the note itself then contains a relative path. Nextcloud Web show the image, the Android App doesn't show it.

![image.png](.attachments.$ID/image.png)

A consistent way to use images in notes regardless of created in Nextcloud Web or Android App or directly by editing .md files and uploading images to the notes folder is highly appreciated. Thanks in advance.

**Comment by stefan-niedermann** [comment]
Just as a quick note here: It does make a difference which editor you are using (see Notes Android Apps settings). They both expect images to be under a different path which is the root cause of images not being displayed.

**Comment by hgdrn** [comment]
Thanks, Stefan for the explanation. In fact I wasn't aware of this.

Unfortunately that makes it even worse (from the user's point of view). So there's an edit mode, a rich edit mode and a preview mode (in both Web and App). And as a quick check I think there's not a single combination of (Web/App and preview/edit/rich edit) where an image is always displayed and is not replaced by a placeholder oder a HTTP 404 error image. An image might be OK "here" but invisible "there" and vice versa. Not complaining, just noticing.

**Comment by stefan-niedermann** [comment]
This is correct and should have been adjusted while integrating Nextcloud Text ("Rich edit mode") into the app. It is probably easier to fix it in the "old" edit mode given Nextcloud Text is used heavily across the Nextcloud ecosystem while the old edit mode is only used in the Notes app (afaik).

**Comment by harrysch** [comment]
Still the same thing: Visible on Web, but not on Android...

**Comment by numeropi** [comment]
I just synced my notes with my laptop: Nextcloud sync GEDIT/Markdown preview plugin  and I realized that markdown or html notes are not working in this awesome FOSS Android client... thanks for checking it out.

**Comment by Malachiel87** [comment]
Any news about it

**Comment by xstable** [comment]
Came here as I have the same issue.

**Comment by sjaaklaan** [comment]
Any news on this? I would like to use the Notes app on my Android phone to include pictures of screens and whiteboards during presentations. This worked perfect in MS OneNote, and it would be very helpful if it worked in the Android Notes app as well.

**Comment by brianjmurrell** [comment]
Is fixing this really as simple as the [commit](https://github.com/amalg/notes-android/commit/89ffb254ca28b80f807b52a82af1d21bc9f262c4) above that is referencing this issue?

I am doubtful, but if it is, why has this not been landed anywhere?

**Comment by merlin3of5** [comment]
I believe i am having the same issue. I can upload images to the web based notes page, but they do not display on the android notes app. I also noticed that I cannot add images in the android notes app.

**Comment by linuxthrawn** [comment]
Same issue. Images added in the browser from "Upload from computer" are visible in the browser. The android app indicates that there are objects in the notes, but pictures do not load at all.
